target/common/test/verilator_lib.cc

```diff
@@ -106,6 +117,47 @@ void tb_memory_write(long long addr, int len, const svOpenArrayHandle data,
                    (const uint8_t *)strb_ptr);
 }
 
+int get_int_from_env(const char* var_name, int default_value = 0) {
+    const char* env_var = std::getenv(var_name);
+    if (env_var != nullptr) {
+        try {
+            // Convert string to integer using std::stoi
+            return std::stoi(env_var);
+        } catch (const std::invalid_argument&) {
+            std::cerr << "Environment variable " << var_name << " is not a valid integer: " << env_var << '\n';
+        } catch (const std::out_of_range&) {
+            std::cerr << "Environment variable " << var_name << " is out of integer range: " << env_var << '\n';
+        }
+    }
+    return default_value; // Return the default value if not found or invalid
+}
+
+svBit enable_tracing() {
```

Comment:
I changed this to svBit, because verilator seemed to generate a DPI function that expected that... not sure how else I would've figured out...